workflow/rules/process/split-world.smk

```diff
@@ -0,0 +1,21 @@
+"""Split the world into boxes
+
+"""
+all_box_geoms = expand(
+    os.path.join(config['data_dir'], "processed", "all_boxes", "{box_id}", "geom_{box_id}.gpkg"),
+    box_id=all_boxes,
+)
+
+
+rule world_splitter:
```

Comment:
I just love the name of this rule. It sounds like something out of Norse mythology.